src/app/my-sidebar/my-sidebar.component.html

```diff
@@ -72,9 +65,12 @@
   </button>
   <hr routerLink="/accounts" routerLinkActive="active" />
 
+  <!-- User Guide -->
```

Comment:
Think docs.dockstore.org may be more of a user guide, the quick start is really just a quick-start. Since we have docs up in the header, think this should say "Quickstart"